src/mozanalysis/bootstrap.py

```diff
@@ -21,6 +21,24 @@ def _confidence(data, confidence_level):
     }
 
 def bootstrap(sc, data, stat_fn, num_iterations=2000, confidence_level=0.95):
+    """Returns a bootstrapped confidence interval
+
+    Given an array of data, returns the bootstrapped confidence interval based
+    on the confidence level provided. This uses pyspark and distributes the data
+    to the nodes for efficiency.
```

Comment:
I'd note the limitations: all the data plus one replicate should be able to fit in memory